frontend/src/features/settings/account/PasswordForm.tsx

```diff
@@ -92,7 +90,6 @@ const PasswordForm = () => {
             label={t('settings.confirmNewPassword')}
             name="password_confirm"
             color="secondary"
-            size="small"
```

Comment:
Why removing `size="small"` specifically on this page ?  
This introduces some inconsistencies between pages, as all other form fields in the app use `size="small"`. 

And I tend to prefer small text fields, to make forms easier to fit on a single screen, and with a more native look-and-feel; although it's mostly a matter of taste as the design of these forms is not currently specified,